### DISCUSSION TITLE
Translations for exclusion grounds (BT-67a+b)

### DISCUSSION THREAD

**Discussion by ghost** [OP-TED/eForms-SDK#533] [Q&A] [answered]
Both fields for exclusion grounds field|name|BT-67(a)-Procedure and field|name|BT-67(b)-Procedure have the same label "Description" in fields_XX.json. I think the labels should be different, e.g. "Code" and "Description"

**Comment by rouschr** [comment, marked as answer]
Thank you, I informed our team and we will look into this.
